### PR TITLE
Add header guards to UniqueElectrons.h

### PR DESCRIPTION
**Pull request by Teemperor**

### CODE REVIEW

**Comment by Teemperor** [comment]
Part of the work going on regarding the C++ modules migration of CMSSW (tracked as issue https://github.com/cms-sw/cmssw/issues/15248). This PR is not meant to refactor things, but just to make these headers compile.

**Comment by davidlange6** [comment]
please test

File: ElectroWeakAnalysis/ZEE/interface/UniqueElectrons.h [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#ifndef UniqueElectrons_h
```
Comment (davidlange6) [inline ElectroWeakAnalysis/ZEE/interface/UniqueElectrons.h]: as elsewhere, please add subsystem and package names to include guards

**Comment by davidlange6** [comment]
-1
pending requested changes

**Pull request closed without merging**